x-pack/plugin/shutdown/src/internalClusterTest/java/org/elasticsearch/xpack/shutdown/NodeShutdownShardsIT.java

```diff
@@ -349,37 +349,15 @@ public void testAutoExpandDuringRestart() throws Exception {
         );
 
         final String nodeB = internalCluster().startNode();
-        assertBusy(() -> {
-            assertThat(
-                client().admin()
-                    .indices()
-                    .prepareGetSettings("myindex")
-                    .setNames("index.number_of_replicas")
-                    .get()
-                    .getSetting("myindex", "index.number_of_replicas"),
-                equalTo("1")
-            );
-        });
         ensureGreen("myindex");
+        var assertion = new BackgroundAssertion(() -> assertIndexSetting("myindex", "index.number_of_replicas", "1"));
```

Comment:
Can we keep the assertions between each step instead? Does not seem like this kind of 100% cpu hog validation is needed here.